### ISSUE TITLE
Spike: Determine the Maximum Volume of Data that can be Processed

### ISSUE THREAD

**Issue by joshmcintosh** [Ross-Alexandra/SENG-371-Project-2#42]
Blocked by GH-34. 
Low Priority.

Given the large volume of data that this project aims to address, it will be necessary to determine the maximum limit that the system can handle so that it can safely cancel the build beforehand.

### Acceptance Criteria

- General numbers for the following:

   - The maximum total volume of data that can be processed

   - The maximum file size that can be processed
